src/openms/include/OpenMS/CHEMISTRY/Ribonucleotide.h

```diff
@@ -170,7 +171,13 @@ namespace OpenMS
     /// Return true if this is a modified ribonucleotide and false otherwise
     bool isModified() const;
 
+    /// Return whether this is an "ambiguous" modification (representing isobaric modifications on the base/ribose)
+    bool isAmbiguous() const;
```

Comment:
can you add this to the pyopenMS .pxd file?

Reply:
I've done this. I'll submit it in a PR once I figure out the other two pxd related things.